app/javascript/packs/components/RemoveMultiple.js

```diff
@@ -15,6 +15,8 @@ function setupClickHandler() {
 
     e.preventDefault();
     removeMultiple(e.target.attributes['data-multiple-remove'].value)
+    showButton()
+    removeMultipleRespondentPTag()
```

Comment:
I know it doesn't seem important - but take another look at the name of this function.  Imagine you are someone new coming to the code base and you are looking at the page in your browser.  Does the name 'removeMultipleRespondetPTag' mean anything to you ?  it doesn't to me.

My guess is that you want to name it based on what the content you are removing actually is ?

Reply:
Also, RemoveMultiple.js is not specific to multiple respondents - I believe it is used for both respondents and claimants - so unsure this is even the right place for it ?